Engine/Module/ModuleUI.h

```diff
@@ -13,6 +39,26 @@ class ModuleUI : public Module
 	bool Init() override;
 	update_status Update() override;
 	bool CleanUp() override;
+
+	void Render(const ComponentCamera* camera);
+
+	ComponentUI* CreateComponentUI(const ComponentUI::UIType, GameObject*);
+	void RemoveComponentUI(ComponentUI*);
+
+	//Glyph init 
+	void InitGlyph();
+	bool glyphInit = false;
+	FT_Library ft;
+	FT_Face face;
+	std::map<GLchar, Character> Characters;
+
+	ComponentCanvas* main_canvas = nullptr;
+	unsigned int text_texture;
+	std::vector<ComponentUI*> ui_elements;
+	float window_width, window_height;
+private:
```

Comment:
This should be splitted using private and public declarations, for more info read the [code guidelines](https://github.com/OnionGalaxy/LittleOrionEngine/blob/master/CONTRIBUTING.md).